helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/model/MessageProxy.java

```diff
@@ -11,7 +11,7 @@
  */
 public class MessageProxy {
```

Comment:
I believe this class no longer required.

Reply:
ok